miden-crypto/src/word/macros.rs

```diff
@@ -9,17 +9,17 @@ use super::{Felt, StarkField};
 #[macro_export]
 macro_rules! word {
     ($hex:expr) => {{
-        let felts: [$crate::Felt; 4] = match $crate::word::parse_hex_string_as_word($hex) {
+        let word: Word = match $crate::word::parse_hex_string_as_word($hex) {
             Ok(v) => v,
             Err(e) => panic!("{}", e),
         };
 
-        $crate::Word::new(felts)
+        word
     }};
 }
 
-/// Parses a hex string into a `[Felt; 4]` array.
-pub const fn parse_hex_string_as_word(hex: &str) -> Result<[Felt; 4], &'static str> {
+/// Parses a hex string into a [`Word`] array.
+pub const fn parse_hex_string_as_word(hex: &str) -> Result<Word, &'static str> {
```

Comment:
Would it make sense to make this a method on the `Word` struct? Something like:
```rust
impl Word {

    pub const fn parse_from_hex_string(hex: &str) -> Result<Self, &'static str> {
        ...
    }
}
```
This would let us avoid importing this function separately (as we currently do in `miden-base`).

Reply:
Makes sense. I've moved the fn to `Word::parse()` now LMK any further changes you would like.